code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -16,8 +16,10 @@
 	var/obj/item/tank/holding
 	///Volume (in L) of the inside of the machine
 	var/volume = 0
-	///Used to track if anything of note has happen while running process_atmos()
-	var/excited = TRUE
+	///Used to track if anything of note has happen while running process_atmos(). 
+	///Treat it as a process_atmos() scope var, we just declare it here to pass it between parent calls.
+	///Should start as false on every process by the way.
```

Comment:
Comments describe what is not what was (much to my bewailing)
If you're going to mention starting on false, you should explain why

Reply:
hope this is adequate